Sekoia.io/sekoiaio/triggers/base.py

```diff
@@ -55,10 +56,13 @@ def liveapi_url(self):
         # from the base URL (which is mandatory).
         if liveapi_url is None:
             base_url = self.module.configuration["base_url"]
-            if "app.test.sekoia.io" in base_url:
+            domain = urlparse(base_url).netloc
+            if domain in ["app.test.sekoia.io", "api.test.sekoia.io"]:
                 liveapi_url = "wss://live.test.sekoia.io/"
-            else:
+            elif domain in ["app.sekoia.io", "api.sekoia.io"]:
                 liveapi_url = "wss://live.sekoia.io/"
+            else:
+                liveapi_url = f"wss://{domain}/live"
```

Comment:
`wss://app.sekoia.io/live` and `wss://app.test.sekoia.io/live` should work fine. Maybe we could fallback on `wss://{domain}/live` in all situations.